satzung.md

```diff
@@ -63,88 +63,102 @@ Die Organe der Fachschaft sind:
 (3) Die in (2) festgelegten Ämter können durch Beschluss mit einer Zwei-Drittel-Mehrheit auch innerhalb einer Amtszeit neu vergeben werden.
 
 
-## §5b. Aufgabenverteilung
+## §4c Aufgabenverteilung
 
-(1) Der Vorsitzende des Fachschaftsrates ist verantwortlich für die Einberufung der Sitzungen, Verwaltung der Tagesordnung und Festlegung der Protokollführung. Er ist ohne weiteren Beschluss automatisch ebenfalls Beauftragter für die Vernetzung mit den anderen Organen der Studierendenschaft laut Satzung der Studierendenschaft der Universität Potsdam.
+(1) Der Vorsitzende des Fachschaftsrates ist verantwortlich für die Sitzungsleitung gemäß §4e. Er ist ohne weiteren Beschluss automatisch ebenfalls Beauftragter für die Vernetzung mit den anderen Organen der Studierendenschaft im Sinne der Satzung der Studierendenschaft der Universität Potsdam.
 
-(2) Der Finanzreferent verwaltet die Finanzen der Fachschaft gemäß den Beschlüssen des Fachschaftsrates. Er und der ggf. gewählte Stellvertretende sind zu einzeln Verfügungsberechtigten zu erklären. Der Finanzreferent sollte einmalig den Finanz-Workshop des Allgemeinen Studierendenausschusses der Universität Potsdam besuchen, sofern dies nicht aufgrund vorhandenen Vorwissens unnötig erscheint.
+(2) Der Finanzreferent verwaltet die Finanzen der Fachschaft gemäß den Beschlüssen des Fachschaftsrates. Er und der ggf. gewählte Stellvertretende sind zu einzeln Verfügungsberechtigten zu erklären. Der Finanzreferent sollte den Finanz-Workshop des Allgemeinen Studierendenausschusses der Universität Potsdam besuchen.
 
 (3) Alle weiteren Kompetenzbereiche können intern durch Beschluss festgelegt werden.
 
 
-## §5c. Anwesenheit
+## §4d Anwesenheit
 
-(1) Bei der Anwesenheit von Mitgliedern des Fachschaftsrates während seiner Sitzungen gilt es zu unterscheiden:
+Bei der Anwesenheit von Mitgliedern des Fachschaftsrates während seiner Sitzungen gilt es zu unterscheiden:
 
 1. persönliche Anwesenheit: Alle Mitglieder des Fachschaftsrates, die sich während der Sitzung am Ort der Sitzung befinden, gelten als persönlich anwesend.
 2. Anwesenheit: Alle Mitglieder des Fachschaftsrates, die während der Sitzung persönlich anwesend sind oder per Telefon oder anderweitiger Einrichtung zur Kommunikation, insbesondere Internet- oder Videotelefonie, teilnehmen, gelten als anwesend.
 
-(2) Während jeder Sitzung muss ein persönlich anwesendes Mitglied den Vorsitz haben. Sind weder der gewählte Vorsitzende noch der Stellvertretende persönlich anwesend, können diese einstimmig den Vorsitz für die betreffende Sitzung auf ein anderes Mitglied übertragen. Diese Entscheidung erhält per E-Mail an den gesamten Fachschaftsrat Gültigkeit.
+
+## §4e Sitzungsleitung
+
+(1) Während jeder Sitzung muss ein persönlich anwesendes Mitglied die Sitzungsleitung innehaben.
```

Comment:
In (2) geht es um "den Sitzungsleitenden" und nicht "die Sitzungsleitenden". Ich würde also sagen, es ist implizit genau einer.